jdbcSource/build.gradle

```diff
@@ -84,4 +84,18 @@ dependencies {
     // Used to create a connection pool if asynchronous processing has been specified for publish/query handlers
     compile group: 'com.zaxxer', name: 'HikariCP', version: '3.3.1'
 
+}
+
+// Using gradle-dcompose-plugin to build jdbcSource docker image
+dcompose {
```

Comment:
You can stick in something like:

```
ext {
    if (rootProject.hasProperty('dockerRepo')) {
        dockerRepo = rootProject.getProperty('dockerRepo')
    } else {
        dockerRepo = 'missing/missing'
    }
}
```

to get the docker repo.  Then, change `repository - ...` below to `repository - "$dockerRepo"`

Reply:
Better will be to skip the task if the repo isn't set.  But I'll have to think about that more...